Images_And_Graphics_style_v1.css

```diff
@@ -85,33 +90,45 @@ margin-top: 20px;
 font-size: 19px;
 margin-left: 955px;
 margin-right: 300px;
-{color: #000080; }
+color: #000080; 
 }
 
 .Phone_number {
 	word-spacing: 5px;
-	margin-left: 150px;
+	margin-left: 2px;
+	margin-right: 60px;
 	text-decoration-line: underline;
 }
 
 .School_Address {
-	letter-spacing: 0.3px;
+  letter-spacing: 0; /* Existing style */
+  margin-left: 0;
+  margin-right: 4px;
+}
+
+.School_Address1 {
+	margin-left: 326px;
 }
 
 .Gmail {
 	letter-spacing: 0;
 	text-decoration-line: underline;
-	margin-right: 5px;
+	margin-right: 0;
 }
 
 .map {
 	text-align: center;
 	margin-bottom: 200px;
-	margin-left: 550px;
-	border-bottom: -15px;
+	border-bottom: 15px;
 	outline: 5px solid white;
+	margin-left: 500px;
 }
 
+.image-container {
+    text-align: center;

```

Comment:
I added an image container so I can align the images in the center. This is because I added the code in a validator and it told me I couldn't add a <center> in html and I should so it in css so I made an image container to hold mmy images and center them in the middle.